File: app/src/main/java/anand/android/applockcloneminapi23/utils/AppToast.java
package anand.android.applockcloneminapi23.utils;

import android.content.Context;

/**
 * Created by anandm on 09/02/17.
 */

public class AppToast {

    public static void showToast(Context context, String message) {
        android.widget.Toast.makeText(context, message, android.widget.Toast.LENGTH_LONG).show();
    }

    public static void showDebugToast(Context context, String message) {
        android.widget.Toast.makeText(context, message, android.widget.Toast.LENGTH_LONG).show();
    }

}

File: app/src/main/java/anand/android/applockclone/utils/AppToast.kt
package anand.android.applockclone.utils

import android.content.Context
import android.widget.Toast

/**
 * Created by anandm on 09/02/17.
 */

object AppToast {

    fun showToast(context: Context, message: String) = Toast.makeText(context, message, android.widget.Toast.LENGTH_LONG).show()


    fun showDebugToast(context: Context, message: String) {
        android.widget.Toast.makeText(context, message, android.widget.Toast.LENGTH_LONG).show()
    }

}
